guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -30,3 +33,39 @@ Pulling container images:
 ----
 # podman pull {foreman-example-com}/my-image:<optional_tag>
 ----
+
+Pushing container images to the {Project} container registry:
+
+* To indicate which organization, product, and repository the container image belongs to, use container namespaces.
+* All labels are composed of entirely lowercase characters regardless of actual label capitalization.
+* Pushes follow one of two formats:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push <image> example.com/<organization_label>/<product_label>/<repository_name>[:tag]
+# podman push <image> example.com/id/<organization_id>/<product_id>/<repository_name>[:tag]
+----
+
+* A complete push looks like the following:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push d2c94e258dcb example.com/default_organization/test_product/hello-world:latest
+Getting image source signatures
+Copying blob ac28800ec8bb done   |
+Copying config d2c94e258d done   |
+Writing manifest to image destination
+----
+
+.Limitations on container registries
+* A repository in Katello is created once the content is pushed.
+* If the organization or product does not exist, Katello sends an error telling the user to ensure the proper entities are created first.
+* Subsequent pushes to the same repository updates the contents of that repository.
+* Content is pulled as soon as it is pushed.
+* Content can only be pushed to the {Project} itself.
+* If pushed content needs to be on {SmartProxies} as well, use {SmartProxy} syncing.
+* Podman requires that all characters in the pushed container name contain only lowercase characters.
+* To push content to an organization or product, a user must be able to create repositories.
```

Comment:
I've seen information about user permissions included under a `.Prerequisite` heading in other modules. Can you create a heading like that and move this line there?

Also, how do I know that my user is able to create repositories? Are we talking about Foreman roles/permissions? Which ones?

Reply:
@ianballou Does a user know if they have permissions to create repositories? Specific role or permissions needed? I am pushing a commit to say that a user must have permissions to create repositories in order to push content to an organization or product.

Reply:
A user indeed must have the same permissions necessary to create a repository. For creation, that would be `edit_products`.

Pulling content is the same way as long as the lifecycle environment in question has "unauthenticated pull" turned off (which is default).

Pull permissions: `view_lifecycle_environments`, `view_products`, `view_content_views`.